### PR TITLE
Support OKEx options

### PR DESCRIPTION
**Pull request by olibre**
### Support OKEx options

Hi Bryant. I have move out some features from the big PR about the AsyncConnection refactoring. This is one of the features.

The feature has been successfully tested in an other branch with much more changes.
But this branch has been tested only with `examples/demo_okex.py`.
Please, could you test this?

- [ ] - Tested
- [x] - Changelog updated
- [ ] - Tests run and pass
- [ ] - Flake8 run and all errors/warnings resolved
- [ ] - Contributors file updated (optional)

### CODE REVIEW

File: cryptofeed/exchange/okcoin.py [hunk 1]
```diff
@@ -54,32 +56,42 @@ def __calc_checksum(self, pair):
...
+
+        # Avoid error "Max frame length of 65536 has been exceeded" by limiting requests to some args
+        for chunk in split.list_by_max_items(symbol_channels, 33):
+            LOG.info("%s: Subscribe to %s args from %r to %r", self.id, len(chunk), chunk[0], chunk[-1])
+            request = {"op": "subscribe", "args": chunk}
+            await conn.send(json.dumps(request))
+
+    @staticmethod
+    def instrument_type(pair):
+        dash_count = pair.count("-")
```
Comment (bmoscon) [inline cryptofeed/exchange/okcoin.py:77]: users can specify other pair separators in symbols.py

**Review by bmoscon — CHANGES_REQUESTED** [review]
only thing I'd change is the pair separator which is not always going to be "-"

**Comment by bmoscon** [comment]
@olibre if you dont have the time to resolve that outstanding issue, let me know and I can fix it myself post merge

**Comment by bmoscon** [comment]
actually I went to use the new function you added to utils for something else, so I just merged it and will resolve myself :) thanks for the PR!

**Comment by olibre** [comment]
Sorry Bryant, I was busy (I am still very busy on other projects...)

**Comment by bmoscon** [comment]
@olibre no worries, happens to everyone :)

**Pull request merged**